cbmc/patches/0001-Bound-packet-remaining-length.patch

```diff
@@ -0,0 +1,28 @@
+From c785aa73d869777a8a0ee4f8ff488397a70e0e29 Mon Sep 17 00:00:00 2001
```

Comment:
Is this patch an addition?  Not just adding a prefix?  Do some proofs change because of this addition?

Reply:
Or does this simply replace the original path with a differently named file?

Reply:
It's not an addition, it simply replace the original path with a differently named file.